build.gradle

```diff
@@ -940,6 +940,7 @@ project(':core') {
     implementation libs.jacksonModuleScala
     implementation libs.jacksonDataformatCsv
     implementation libs.jacksonJDK8Datatypes
+    implementation libs.jacksonAfterburner
```

Comment:
Please help me understand why did we add this dependency in clients?

Reply:
I have been misled by other Jackson dependencies. Removed.